Dockerfile

```
@@ -0,0 +1,106 @@
+# Documentation only: EXPOSE neither publishes the port nor can interpolate
+# the runtime HTTP_PORT env var. The optional /healthz + /metrics surface
+# (src/http.rs) only listens when the operator sets HTTP_PORT, and 9090 is
+# the value .env.example documents as the convention. Actual publishing
+# happens in compose via `ports:`.
+EXPOSE 9090
+
+# Liveness check: is the `liquidator` process still running. Deliberately
+# NOT wired to /healthz - that endpoint is a readiness signal (can the bot
+# currently reach the chain?), not a liveness one, and restarting a process
+# stuck because RPC is down doesn't fix RPC being down (see src/http.rs).
+HEALTHCHECK --interval=60s --timeout=10s --start-period=10s --retries=3 \
```

Hadolint (2.15.1)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)